modules/programs/xmobar.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.xmobar;
+in {
+  options.programs.xmobar = {
+    enable = mkEnableOption "Xmobar: a minimalistic status bar";
```

Comment:
I think changing the colon to a comma would read better in context ("Whether to enable Xmobar, a minimalistic status bar").